### PR TITLE
Batch estimation of climbing dynamics

### PR DESCRIPTION
**Pull request by gunturiCM**
## Main Changes
This is an attempt to simulate the discrete-time altitude dynamics of a quadcopter.
This also attempts to estimate the values of `alpha` and `beta` using synthetic data produced through the simulations. This estimation is for testing purposes only. Real altitude data will be collected to replace the synthetic data and true values of `alpha` and `beta` will be calculated.

## Tests
The estimated values are 
```
alpha hat = 0.994749 
c hat = -9.449992
```
which are close to the original values 
```
alpha = 1 
c = -9.81
```
The simulation results are plotted and shown below. 
least_sqs_altitude_params_estimation

### CODE REVIEW

File: simulations/estimate_SS_parameters_by_LSM.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# TODO: Documentation
```
Comment (alphaville) [inline simulations/estimate_SS_parameters_by_LSM.py:1]: There's a TODO here

**Review by alphaville — APPROVED** [review]
This looks good. I only made a minor comment. I believe we can merge this. We could also implement the same thing using a Kalman filter and see how the KF estimates converge in time. It would be a useful exercise.

**Comment by alphaville** [comment]
@gunturiCM What is the status of this PR?

**Comment by gunturiCM** [comment]
Finished the project in #107. Closing this PR since it is not needed anymore.

**Pull request closed without merging**